lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -211,6 +220,7 @@ def safe_bulk(actions)
         sleep_interval = next_sleep_interval(sleep_interval)
         retry unless @stopping.true?
       rescue ::LogStash::Outputs::ElasticSearch::HttpClient::Pool::BadResponseCodeError => e
+        safe_url = ::LogStash::Outputs::ElasticSearch::SafeURL.without_credentials(e.url)
```

Comment:
Can you rebase this on master? Luckily we no longer need this as of today's merge of #450, `e.url` is now a `LogStash::Util::SafeURI`. For extra safety you can now log `e.url.sanitize`.